synopsys-task/src/synopsys-task/azure-service-client.ts

```diff
@@ -14,10 +15,10 @@ export class SynopsysAzureService {
   async getPullRequestIdForClassicEditorFlow(
     azureData: AzureData
   ): Promise<number> {
-    if (
-      process.env["BUILD_REASON"] &&
-      process.env["BUILD_REASON"] !== "PullRequest"
-    ) {
+    taskLib.debug(":::getPullRequestIdForClassicEditorFlow:::");
+    const buildReason = process.env["BUILD_REASON"];
+    taskLib.debug("buildReason:::" + buildReason);
+    if (buildReason && buildReason !== AZURE_BUILD_REASON.PULL_REQUEST) {
```

Comment:
Do we need this logic?
I think we are calling this method only if it PR event is true and BD/Coverity PR comment is enabled
Verify this once, if required reuse the isPullRequest() method from utility